cfpb/cfgov-refresh
# Update to Capital Framework 8

This is primarily meant to [update cf-forms and fix a bug with our large-target radio buttons and checkboxes in IE](https://github.com/cfpb/capital-framework/pull/900), but since I was in there, I confirmed that the entire framework could be updated safely to v8, since it's just an internal update for all the rest of the components, with no user-facing changes.

## Changes

- Update all Capital Framework packages to the latest

## Testing

1. Pull branch
1. `./frontend.sh`
1. View http://localhost:8000/about-us/blog/ in IE via Sauce or Virtual Box and see that the checkboxes in the filter controls render properly

## Notes

- I am not sure why `cfgov/unprocessed/apps/hmda/yarn.lock` was created. Should I remove it?
- Hmm... I think I should've undone the changes in the `.yarnrc` file, too. I will tidy that up tomorrow.
- I do not understand why the older binaries for cf-core, cf-grid, cf-layout, and cf-typography were not deleted. References to them are still hanging around in the `yarn.lock` file, too.

## Checklist
